### ISSUE TITLE
SoftAP running ancient TKIP encryption. iPhones complain.

### ISSUE THREAD

**Issue by etherfi** [espressif/arduino-esp32#4805]
### Hardware:
Board:							ESP-WROOM-32 Dev Board (FCC ID: 2AC7Z-ESPWROOM32)
Core Installation version:			1.0.5-rc6
IDE name:						Arduino IDE
Flash Frequency:					80Mhz
PSRAM enabled:                                     N/A
Upload Speed:					115200
Computer OS:                                        Windows 10

### Description:
When connecting to a SoftAP created by the sketch below from my iPhone, I receive this message:

**Weak Security**
WPA/WPA2 (TKIP) is not considered secure.
If this is your Wi-Fi network, configure the router to use WPA2 (AES) or WPA3 security type.

How do I fix this?

### Sketch:
```cpp
#include <WiFi.h>

void setup() {
  WiFi.softAP("Test-AP", "testpass");
}

void loop() {
  // put your main code here, to run repeatedly:

}
```

### Debug Messages:
```
N/A
```

**Comment by etherfi** [comment]
Bump

**Comment by me-no-dev** [comment]
encryption is set here: https://github.com/espressif/arduino-esp32/blob/master/libraries/WiFi/src/WiFiAP.cpp#L127

options available in this version of esp-idf (v3.3) are here: https://github.com/espressif/arduino-esp32/blob/master/tools/sdk/include/esp32/esp_wifi_types.h#L55-L63

For adding support for other types of encryption, you can post an issue in esp-idf's issue tracker.
